docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,13 @@
-<!-- ❤️ Thanks for filing a Pull Request on Calva! You are contributing to a better Clojure coding experience. ❤️
+<!--
+❤️ Thanks for filing a Pull Request on Calva! You are contributing to a better Clojure coding experience. ❤️
+
+PLEASE NOTE:
+If you want to file a Pull Request on the documentation of Calva (calva.io),
+then use the Documentation PR template by adding 'template=docs.md' to the
+query parameters of the URL of this page.
```

Comment:
I wonder about this adding more friction, even if it's a small ask, but we'll see if it's problematic. I like having a separate template, anyway.